src/plugins/vis_types/timeseries/public/application/visualizations/views/annotation.js

```diff
@@ -9,7 +9,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import moment from 'moment';
-import reactcss from 'reactcss';
```

Comment:
please remove that lib from package.json too

Reply:
done